website/docs/getting-started/setup/installation-guides/air-gapped.md

```diff
@@ -14,7 +14,7 @@ Privacy-minded users may equate air-gapping with the reluctance to share usage t
 
 ## Whitelisting and requesting Airgapped Edition
 
-The majority of users can effectively use Appsmith by allowing access to `cs.appsmith.com`. However, in organizations where maintaining complete internet isolation is a critical compliance requirement, obtaining an airgapped edition is imperative. To initiate this process, contact the Appsmith sales team at `sales@appsmith.com`.
+The majority of users can access Appsmith by allowing the domain `cs.appsmith.com`. When whitelisting, ensure that you add the domain `cs.appsmith.com`, as IP-based whitelisting is not allowed for this domain because it relies on dynamic IPs for scaling and load balancing. However, in organizations where maintaining complete internet isolation is a critical compliance requirement, obtaining an airgapped edition is imperative. To initiate this process, contact the Appsmith sales team at `sales@appsmith.com`.
```

Comment:
deploy may be ?

Reply:
This is after deploying -- so just access.